app/src/api/index.ts

```diff
@@ -43,7 +44,11 @@ export const subscribe = async (subject: string, state: Ref<Record<string, Model
   opts.deliverAll();
   opts.ackNone();
   opts.replayInstantly();
-  const sub = await js.subscribe(subject, opts);
+  const sub = await js.value.subscribe(subject, opts);
+  if (subscriptions.value[subject] !== 'pending') {
+    sub.unsubscribe();
+    return;
+  }
```

Comment:
Why is this a thing? Wouldn't the call be blocked by line 32 to 35 already? And why do you unsubscribe then?

Reply:
@anbraten I just wrote a test for this case: [fb22be8](https://github.com/kiel-live/kiel-live/pull/210/commits/fb22be8ecbf0241f10518539a288a62c40ea0909)

Reply:
```suggestion

  // If a subscription got unsubscribed before the subscription process finished immediately unsubscribe at this place again
  if (subscriptions.value[subject] !== 'pending') {
    sub.unsubscribe();
    return;
  }
```